bundles/redhat8.6/bundle.sh.gotmpl

```diff
@@ -49,6 +49,7 @@ subscription::defer_unregister
 ENABLED_REPOS="kubernetes,codeready-builder-for-rhel-8-x86_64-rpms,rhel-8-for-x86_64-appstream-rpms,rhel-8-for-x86_64-baseos-rpms"
 EUS_REPOS=${EUS_REPOS:-""}
 if [[ -n "${EUS_REPOS}" ]]; then
+  subscription-manager repos --disable=rhel-8-for-x86_64-baseos-rpms --disable=rhel-8-for-x86_64-appstream-rpms #disable baseos repos
```

Comment:
nit: Can you please explain what this does in more details, and make it more prominent? 

(Does this remove the repos for the "current" RHEL release? :thinking:)

Reply:
i added a note that explains what it does and why.